cmd/src/campaign_plans.go

```diff
@@ -0,0 +1,167 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+var campaignPlansCommands commander
+
+func init() {
+	usage := `'src campaign plans' creates and previews campaign plans (which can be used to create campaigns and changesets).
+
+EXPERIMENTAL: Campaigns are experimental functionality on Sourcegraph and in the 'src' tool.
+
+Usage:
+
+	src campaign plans command [command options]
+
+The commands are:
+
+	preview           generates preview of plan from a specification
```

Comment:
I wonder whether this shouldn't be called `create` since, technically, you _create the plan_ and _preview the changesets_.